packages/flutter_blue_plus_linux/lib/flutter_blue_plus_linux.dart

```diff
@@ -660,6 +655,7 @@ final class FlutterBluePlusLinux extends FlutterBluePlusPlatform {
           remoteId: request.remoteId,
           serviceUuid: request.serviceUuid,
           characteristicUuid: request.characteristicUuid,
+          characteristicIndex: 0,
```

Comment:
Should this be the `characteristicIndex` from the `request` instead of `0`?

```suggestion
          characteristicIndex: request.characteristicIndex,
```

Reply:
Same comment as previous.  The index is only used for initiating operations.  It is not returned.

Reply:
> It is not returned in the returned data structures.

If it is unused and not returned then should it be removed from the `BmDescriptorData` model?